src/components/WebcamCapture/WebcamCapture.test.js

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
 import {shallow, configure} from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
-import WebcamCaptureContainer from './WebcamCaptureContainer';
+import WebcamCapture from './WebcamCapture';
 
 configure({adapter: new Adapter()});
 
-it('It renders a webcam capture container correctly', () => {
-	const wrapper = shallow(<WebcamCaptureContainer />);
-	expect(wrapper).toHaveLength(1);
+it("Tells the user that webcam isn't found if not found", () => {
+	const wrapper = shallow(
+		<WebcamCapture cameraConnected={false} cameraRef={{current: null}} />
+	);
+	expect(wrapper.text()).toEqual('Cannot access camera');
+});
+
+it('Renders webcam with correct props if webcamconnected', () => {
```

Comment:
previous test is good -lets discuss this one though

Reply:
Ive updated the test